addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2838,10 +2861,10 @@ we-select.o_grid we-toggler {
 
         .o_snippet_preview_wrap {
             min-height: 100px;
-            transition: all 150ms cubic-bezier(0.65, 0.05, 0.36, 1);
-            box-shadow: 0 0.5rem 6rem rgba(0, 0, 0, 0.2);
-            margin-bottom: 96px !important;
+            box-shadow: 0 0 6rem rgba(0, 0, 0, 0.16);
```

Comment:
It's strange but, on Chrome, with this change the "embed code" preview is blurred

Reply:
I can't replicate, strange indeed 🤔
Are you sure that you're not referring the the shadow that the code-editor window has in the picture itself?
![image](https://github.com/user-attachments/assets/7fd9b7f3-9e5b-42e1-b41f-fe3d31f3161e)

Reply:
no look at my video

https://github.com/user-attachments/assets/ac93cf38-d25a-4938-809c-05821b1811c2